### PR TITLE
update to release-signed driver

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Updates the included NPM driver to the release-signed driver

### CODE REVIEW

**Review by brycekahle — APPROVED** [review]

**Review by brycekahle — APPROVED** [review]

**Review by leeavital — APPROVED** [review]

File: pkg/network/ddnpmapi.h [hunk 1]
```diff
@@ -122,7 +122,7 @@ typedef struct _filterAddress
     uint8_t                   v4_address[4];    // address in network byte order, so v4_address[0] = top network tuple
     uint8_t                   v4_padding[4];    // pad out to 64 bit boundary
     uint8_t                   v6_address[16];
-    uint64_t                  mask; // number of mask bits.
+    uint64_t                  mask; // number of mask bits.
```
Comment (albertvaka) [inline pkg/network/ddnpmapi.h:125]:
```suggestion
    uint64_t                  mask; // number of mask bits.
```

File: pkg/network/ddnpmapi.h [hunk 2]
```diff
@@ -161,7 +161,7 @@ typedef struct _filterDefinition
  * PACKET_HEADER structure
  *
  * provided by the driver during the upcall with implementation specific
- * information in the header.
+ * information in the header.
```
Comment (albertvaka) [inline pkg/network/ddnpmapi.h:164]:
```suggestion
 * information in the header.
```

File: pkg/network/ddnpmapi.h [hunk 3]
```diff
@@ -186,7 +186,7 @@ typedef struct _perFlowData {
     uint8_t           localAddress[16];  // only first 4 bytes valid for AF_INET, in network byte order
     uint8_t           remoteAddress[16]; // ditto
 
-    // stats common to all
+    // stats common to all
```
Comment (albertvaka) [inline pkg/network/ddnpmapi.h:189]:
```suggestion
    // stats common to all
```

File: pkg/network/ddnpmapi.h [hunk 4]
```diff
@@ -240,6 +240,7 @@ typedef struct filterPacketHeader
     uint64_t        pktSize;                //! size of packet
     uint64_t        af;		                //! address family of packet
     uint64_t        ownerPid;               //! (-1) if not available
+    uint64_t        timestamp;              // timestamp in ns since unix epoch
```
Comment (albertvaka) [inline pkg/network/ddnpmapi.h:243]: Unused?

**Review by albertvaka — APPROVED** [review]
Remove the trailing whitespaces that were added before merging.

File: pkg/network/ddnpmapi.h (hunk 4 above)
Comment (brycekahle) [reply pkg/network/ddnpmapi.h:243]: This will be used, and was the reason for the update.

Comment (derekwbrown) [reply pkg/network/ddnpmapi.h:243]: it will be; that was the whole point of the driver PR.

**Pull request merged**